frontend/packages/console-app/src/components/user-preferences/UserPreferencePage.scss

```
@@ -0,0 +1,16 @@
+.co-user-preference-page-content {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--pf-global--BackgroundColor--100);
+  height: 100%;
+  padding: 0 var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) 0;
+  &__tab-content {
+    margin: var(--pf-global--spacer--lg) 0 0 var(--pf-global--spacer--3xl);
```

The content width is already depending on the description length (help text below the select)

I believe we should give all content elements the full width instead. If you or UX doesn't like that we can skip `flex: 1` here. But with the simplified padding above the margin could be simplified here as well:

```suggestion
    flex: 1;
    margin: var(--pf-global--spacer--lg);
```

---

As discussed, we could wrap the description instead, similar to what is done in the import forms.